### DISCUSSION TITLE
How to assess PTM localization confidence?

### DISCUSSION THREAD

**Discussion by Munchic** [vdemichev/DiaNN#427] [Q&A]
Hello, I am trying to compare confidence of phospho localization in a few DIA-NN settings. So far, I added the following additional options: `--monitor-mod UniMod:1 --monitor-mod UniMod:21 --monitor-mod UniMod:4`

A few questions:
1. What is the difference between "PTM.Site.Confidence", "Lib.PTM.Site.Confidence", "PTM.Localising" and which one should I use for my purpose?
2. When adding --monitor-mod, that score is automatically incorporated in the above metrics and there is no separate column for each modification's confidence?
3. In general, how would you suggest comparing DIA-NN outputs from two sets of parameters? E.g., searching 2 var mods vs. 3 var mods.

Thank you so much for your help and guidance!

**Comment by vdemichev** [comment]
Hi,

1. Lib. values are those recorded during the library creation stage. Whether it's OK to use them instead of run-specific ones probably depends on the experiment. PTM.Localising is not supposed to be used.
2. DIA-NN reports localisation confidence for all sites on the peptide combined.
3. Number of modified peptides at PTM.Q.Value < 0.01.

Best,
Vadim

**Reply by Munchic** [reply]
Thank you, this is helpful!

**Reply by sooheon** [reply]
Hi vadim, the docs now say that ptm-qvalue is "legacy", does this mean for phosphosite filtering i should rely only on lib.q.value and ptm.site.confidence, and not ptm.q.value?

**Reply by vdemichev** [reply]
Hi Sooheon,

Please just use Peptidoform.Q.Value instead of PTM.Q.Value.

Best,
Vadim

**Reply by lori-koch** [reply]
Hi Vadim,
I am using the latest release DIANN 1.9.2 however I only see PTM.Q.Value etc as column names in my report.tsv and not Peptidoform.Q.Value. What is the difference? How can I get the "peptidoform" scores to do the thresholding as described above? Thanks

**Reply by vdemichev** [reply]
Hi, 

Please use the .parquet report.
The .tsv report is only for compatibility with old analysis scripts and therefore does not contain any new functionality.

Best,
Vadim

**Comment by haiyan-MS** [comment]
HI Vadim: 

I am also having the same issue. I am using 1.9.2, but I don't see "peptidoform" scores in report.tsv. You said we should use .parquet report. But I don't know how to view the parqeut files.

**Reply by vdemichev** [reply]
Please see https://github.com/vdemichev/DiaNN?tab=readme-ov-file#output & expand the 'Main report'.
